src/main/resources/schema/system/integration.schema

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
this file already exists?

Reply:
Yeah, had to copy it to the resources path to get it to work in the moment, will need to find an alternative way to get global paths later.